lib/main.js

```diff
@@ -199,9 +199,13 @@ export default {
         } else {
           // We must determine this ourself for lower versions
           const baseName = path.basename(filePath);
-          if (ignorePatterns.some(pattern => minimatch(baseName, pattern))) {
-            return [];
-          }
+          const dirName = path.dirname(filePath);
+
+          if (ignorePatterns.some((pattern) => {
+            return minimatch(baseName, pattern) ||
+              minimatch(dirName, pattern) ||
+              minimatch(filePath, pattern);
+          })) return [];
```

Comment:
Put this on a new line, and wrap it in `{}`, I'm a bit surprised the lint rules aren't throwing a fit over this...